gnovm/tests/files/zrealm_crossrealm24f.gno

```diff
@@ -0,0 +1,16 @@
+// PKGPATH: gno.land/r/crossrealm_test
+package crossrealm_test
+
+import (
+	"gno.land/p/demo/tests"
+)
+
+var root interface{}
+
+func main() {
+	root = tests.F // already attached in p
```

Comment:
```go
	// Should succeed, tests.F should be copied by value
	root = tests.F
	// Should succeed, tests.F should be referenced, immutable, cannot modify
	// root.(*tests.Foo).name (even if exported) because it's a value "attached"
	// to a p/ package
	root = &tests.F
```

Reply:
done